lessons/taquito/04/04.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Michelson is a thing you know.
+chapter: Chapter 4
+slug: module-04/chapter-04
+filterBy: lesson-4
+editor:
+  language: javascript
+  startingCode: |
+    const FAUCET_KEY = {
+        "mnemonic": [
+            "chimney",
+            "item",
+            "course",
+            "pair",
+            "opinion",
+            "mass",
+            "receive",
+            "wrong",
+            "hurry",
+            "nest",
+            "oil",
+            "acoustic",
+            "section",
+            "salad",
+            "electric"
+        ],
+        "secret": "47184b1d8bf4dd550256817c3507cc4ccf112888",
+        "amount": "903205016",
+        "pkh": "tz1Q654L2qnt5mDM1Lbi4rMSTtnE5JHvD8M7",
+        "password": "9U5VOADM6l",
+        "email": "ryqwtuyj.ciwjnvdt@tezos.example.org"
+    }
+    
+    importKey(
+        Tezos,
+        FAUCET_KEY.email,
+        FAUCET_KEY.password,
+        FAUCET_KEY.mnemonic.join(' '),
+        FAUCET_KEY.secret
+    );
+    
+    Tezos.setProvider({ rpc: 'https://carthagenet.SmartPy.io' });
+    
+    // Store the code here - 
+    // const CODE_JSON = 
+    
+    // Store the initial storage over here - 
+    // const STORAGE_JSON = 
+
+  answer: |
+    const FAUCET_KEY = {
+        "mnemonic": [
+            "chimney",
+            "item",
+            "course",
+            "pair",
+            "opinion",
+            "mass",
+            "receive",
+            "wrong",
+            "hurry",
+            "nest",
+            "oil",
+            "acoustic",
+            "section",
+            "salad",
+            "electric"
+        ],
+        "secret": "47184b1d8bf4dd550256817c3507cc4ccf112888",
+        "amount": "903205016",
+        "pkh": "tz1Q654L2qnt5mDM1Lbi4rMSTtnE5JHvD8M7",
+        "password": "9U5VOADM6l",
+        "email": "ryqwtuyj.ciwjnvdt@tezos.example.org"
+    }
+        
+    importKey(
+        Tezos,
+        FAUCET_KEY.email,
+        FAUCET_KEY.password,
+        FAUCET_KEY.mnemonic.join(' '),
+        FAUCET_KEY.secret
+    );
+    
+    Tezos.setProvider({ rpc: 'https://carthagenet.SmartPy.io' });
+    
+    // Store the code here - 
+    // const CODE_JSON = 
+    
+    // Store the initial storage over here - 
+    // const STORAGE_JSON = 
+    
+---
+### Intro
+Michelson is the language than runs on the Tezos network, rest all higher-level languages are an abstraction over it which directly compile to michelson.
+
+### Study time
+
+To deploy contracts through Taquito you need to grab the "Michelson JSON" and "Initial Storage JSON" for the contract that you wrote in SmartPy and this process is made SUPER EASY thanks to the [SmartPy IDE](smartpy.io/dev) - it generates "Storage JSON" and "Code JSON" for you which you can directly copy.
```

Comment:
Doesn't redirect to SmartPy IDE. It's considering it as a relative link. Try adding this: https://smartpy.io/dev/